Sources/Build/BuildPlan.swift

```diff
@@ -211,6 +224,8 @@ public enum TargetBuildDescription {
             return target.libraryBinaryPaths
         case .clang(let target):
             return target.libraryBinaryPaths
+        case .mixed(let target):
+            return target.swiftTargetBuildDescription.libraryBinaryPaths
```

Comment:
Same here. There should be one unified `libraryBinaryPaths`

Reply:
Same explanation.